File: C++/CodeForces/R-611-div.3/pA.go
/************************************/
/* Date		: 2020-01-17 10:06:47	*/
/* Author	: MiohitoKiri5474		*/
/* Email	: lltzpp@gmail.com		*/
/************************************/

// by. MiohitoKiri5474

package main

import (
	"fmt"
)

func main() {
	var h, m, t int
	fmt.Scanf ( "%d\n", &t )
	for t > 0 {
		t--
		ans := 0
		fmt.Scanf ( "%d %d\n", &h, &m )
		ans = ( 24 - h ) * 60
		if m != 0 {
			ans -= 60
			ans += 60 - m
		}
		fmt.Println ( ans )
	}
}

File: C++/CodeForces/R-562-div.2/pA.py
# -*- coding: utf-8 -*-
# Date    : 2019-05-29 19:19:17
# Author  : MiohitoKiri5474 ( lltzpp@gmail.com )
# Link    : https://miohitokiri5474.github.io

n, a, x, b, y = [int(_) for _ in input().strip().split()]
while a != x and b != y:
	if a == b:
		break

	if a != n:
		a += 1
	else:
		a = 1

	if b != 1:
		b -= 1
	else:
		b = n

if a == b:
	print ( "Yes" )
else:
	print ( "No" )